firefox-ios/Client/Frontend/Browser/Toolbars/AddressToolbarContainer.swift

```diff
@@ -126,23 +157,50 @@ class AddressToolbarContainer: UIView,
         compactToolbar.removeFromSuperview()
         regularToolbar.removeFromSuperview()
 
-        let isCompact = traitCollection.horizontalSizeClass == .compact
-        let toolbarToAdd = isCompact ? compactToolbar : regularToolbar
-
-        addSubview(toolbarToAdd)
+        addSubview(toolbarType)
+        toolbarType.addSubview(progressBar)
```

Comment:
I don't think the progress bar should be a subview of the toolbar but instead should be one of the container itself.